From a03c52fbd7df5ce82ddd6e029ba44b08e336803f Mon Sep 17 00:00:00 2001
Message-Id: <a03c52fbd7df5ce82ddd6e029ba44b08e336803f.1423041670.git.chang-joon.lee@intel.com>
In-Reply-To: <db1faedd0784f7fc40add45bbfffd4007c07ab72.1423041670.git.chang-joon.lee@intel.com>
References: <db1faedd0784f7fc40add45bbfffd4007c07ab72.1423041670.git.chang-joon.lee@intel.com>
From: Uma Shankar <uma.shankar@intel.com>
Date: Wed, 21 Jan 2015 20:05:26 +0530
Subject: [PATCH 2/2] SQUASHME!: [VPG]: drm/i915: Removed redundant mode
 changing notification

There was a redundant event for mode changing sent to audio driver during
active video playback. This patch fixes the same.

SQUASHME!: This should be sqaushed with below existing patch
	Author: Vidya Srinivas <vidya.srinivas@intel.com>
	Change-Id: I16152551ca5dd6951824e99984dcfba4d6b42e35
	REVERTME [VPG]: drm/i915: Added modeset notification to HAD.

Issue: IRDA-2856
Change-Id: Ibc141756ee4a1c54de55f65a0dfe326c859c067e
Signed-off-by: Uma Shankar <uma.shankar@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    9 ---------
 1 file changed, 9 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 10c45c7..dafc6a8 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -2687,15 +2687,6 @@ static void i9xx_update_primary_plane(struct drm_crtc *crtc,
 	if (!dev_priv->atomic_update)
 		intel_update_watermarks(crtc);
 
-#ifdef CONFIG_SUPPORT_LPDMA_HDMI_AUDIO
-	if (IS_VALLEYVIEW(dev) && intel_pipe_has_type(crtc,
-		INTEL_OUTPUT_HDMI)) {
-		dev_priv->tmds_clock_speed = intel_crtc->config.port_clock;
-		mid_hdmi_audio_signal_event(dev_priv->dev,
-			HAD_EVENT_MODE_CHANGING);
-	}
-#endif
-
 	reg = DSPCNTR(plane);
 	/*
 	 * In case of atomic update, primary enable/disable is already cached as
-- 
1.7.9.5

